tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/GitStore.cs

```diff
@@ -32,7 +32,11 @@ public class GitStore : IAssetsStore
         public GitProcessHandler GitHandler = new GitProcessHandler();
         public string DefaultBranch = "main";
         public string FileName = "assets.json";
-        public static readonly string EnvironmentVariableName = "PROXY_GIT_TOKEN";
+        public static readonly string GIT_TOKEN_ENV_VAR = "GIT_TOKEN";
+        // Note: These are slightly different from the GIT_COMMITTER_NAME and GIT_COMMITTER_EMAIL
```

Comment:
We should discuss with Wes and Ben as to their opinions on not overloading the defaults. I'm good with merging this though.

Reply:
Variable names are easy to change when this isn't being actively used :)